.github/workflows/cd.yml

```
@@ -0,0 +1,50 @@
+          cache-workspace-crates: true
+
+      - name: Set up toolchain
+        uses: moonrepo/setup-toolchain@v0
+        with:
+          auto-install: true
+
+      - name: Build the app
+        run: moon portfolio:build/release
+
+      - name: Deploy to Github Pages
+        uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.7)

the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)